core/conventional-commits/lib/update-changelog.js

```diff
@@ -8,16 +8,26 @@ const { BLANK_LINE, CHANGELOG_HEADER, EOL } = require("./constants");
 const getChangelogConfig = require("./get-changelog-config");
 const makeBumpOnlyFilter = require("./make-bump-only-filter");
 const readExistingChangelog = require("./read-existing-changelog");
+const getCustomChangeLogHeader = require("./get-custom-changelog-header");
 
 module.exports = updateChangelog;
 
-function updateChangelog(pkg, type, { changelogPreset, rootPath, tagPrefix = "v", version }) {
+function updateChangelog(
+  pkg,
+  type,
+  { changelogPreset, rootPath, tagPrefix = "v", version, changelogHeader: header }
+) {
   log.silly(type, "for %s at %s", pkg.name, pkg.location);
 
   return getChangelogConfig(changelogPreset, rootPath).then(config => {
     const options = {};
     const context = {}; // pass as positional because cc-core's merge-config is wack
-
+    let changelogHeader = CHANGELOG_HEADER;
+    // check if custom change log header is passed
+    if (header && typeof header === "string") {
```

Comment:
I would extract all of this logic into a `getChangelogHeader(title = "Change Log")` method that returns the entire header array with (possibly) customized title value.